docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -63,6 +63,12 @@ Create a .NET console app project named "HelloWorld".
 
    The first time you edit a *.cs* file, Visual Studio Code prompts you to add the missing assets to build and debug your app. Select **Yes**, and Visual Studio Code creates a *.vscode* folder with *launch.json* and *tasks.json* files.
 
+   > [!NOTE]
+   > If you don't get the prompt, or if you accidentally dismiss it without selecting **Yes**, do the following steps to create *launch.json* and *tasks.json*:
+   >
+   >* Select **Run** > **Add Configuration...** from the menu.
```

Comment:
Guidance is here: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=29014

```suggestion
   >* Select **Run** > **Add Configuration** from the menu.
```